### DISCUSSION TITLE
dumi not work when executed `npm start` in windows

### DISCUSSION THREAD

**Discussion by coderbaozi** [ant-design/ant-design#42261] [Q&A]
[![p9DhtjU.md.png](https://s1.ax1x.com/2023/05/10/p9DhtjU.md.png)](https://imgse.com/i/p9DhtjU)

**Comment by zombieJ** [comment]
Pull latest code. It's a bug has been fixed and not related with dumi.

**Reply by coderbaozi** [reply]
<img width="338" alt="image" src="https://github.com/ant-design/ant-design/assets/103836393/47efd354-b57a-4fd3-b437-476de33391dc">

As you can see, I've pulled the latest code from the master branch, and the situation doesn't seem to be getting any better

**Reply by BoyYangzai** [reply]
这个问题 我上周回学校的时候也遇到了 学校电脑是windows 但是我忘记提issue了
我当时看到报错 发现没有生成es下的tokenmeta.json 然后手动跑了构建es文件的脚本 好像是报没有 rmxxx的脚本 因为windows没有rm命令 然后就没有心情管了 总之windows坑挺多的 ovo

**Reply by coderbaozi** [reply]
🐏神你说的对，但我是用gitbash跑的gitbash不是支持linux命令吗

**Reply by zombieJ** [reply]
CC @MadCcc

**Reply by yangxiuxiu1115** [reply]
我今天拉取了最新的代码，也出现了对应的问题。可以问下解决方案嘛。
